net/pfSense-pkg-haproxy-devel/files/usr/local/www/shortcuts/pkg_haproxy.inc

```diff
@@ -24,6 +24,11 @@ global $shortcuts;
 
 $shortcuts['haproxy'] = array();
 $shortcuts['haproxy']['main'] = "/haproxy/haproxy_global.php";
-$shortcuts['haproxy']['log'] = "/status_pkglogs.php?pkg=haproxy-devel";
+$loglink = '../status_pkglogs.php';
```

Comment:
This will break if the page is not in `/`. Use a version test, not a file test, and avoid single-use variables.